conf/base.config

```diff
@@ -11,13 +11,13 @@
 process {
 
     // TODO nf-core: Check the defaults for all processes
-    cpus   = { 1      * task.attempt }
-    memory = { 6.GB   * task.attempt }
-    time   = { 4.h    * task.attempt }
+    cpus = { 1 * task.attempt }
```

Comment:
I see what you've done here, and I'm fine with it. There is a general approach that people seem to like anf that the template generates that the equals sighs are vertical.